core/src/avm1/globals.rs

```diff
@@ -870,7 +872,15 @@ pub fn create_globals<'gc>(
 
     flash.define_value(gc_context, "net", net.into(), Attribute::empty());
 
-    let file_reference_obj = file_reference::create_constructor(
+    let file_reference = file_reference::create_constructor(
+        context,
+        object_proto,
+        function_proto,
+        array_proto,
+        broadcaster_functions,
+    );
+
+    let file_reference_list_obj = file_reference_list::create_constructor(
```

Comment:
Why the `file_reference_obj` -> `file_reference` rename, if then `file_reference_list_obj` is added? Or the other way around...

Reply:
The rename was just for nicer syntax on adding to `SystemPrototypes`, but I'll rename the other for consistency